cartridges/int_bolt_embedded_sfra/cartridge/scripts/services/httpUtils.js

```diff
@@ -44,7 +44,7 @@ exports.restAPIClient = function (
             service.addHeader('X-Nonce', new Date().getTime());
             service.addHeader('X-Bolt-Source-Name', constants.BOLT_SOURCE_NAME);
             service.addHeader('X-Bolt-Source-Version', constants.BOLT_CARTRIDGE_VERSION);
-            if (authenticationHeader) {
+            if (authenticationHeader && authenticationHeader != null) {
```

Comment:
Open to not modifying this, if there is an easier way to not pass in a conditional argument in javascript.

Reply:
I figured it out

Reply:
here if `authenticationHeader` is true then it won't be null

Reply:
That's actually not true in javascript I think. Null and undefined are different, undefined being a singleton class.

Reply:
https://developer.mozilla.org/en-US/docs/Glossary/Falsy

Reply:
nvm I was wrong :)